docs/section4/4-14.md

```diff
@@ -0,0 +1,38 @@
+# Airside System - Rule 4-14  
+**Schema Version:** 0.0.13  
+**Mandatory Rule:** True   
+**Rule ID:** 4-14  
+**Rule Description:** A computer room is defined as a room whose primary function is to house equipment for the processing and storage of electronic data and that has a design electronic data equipment power density exceeding 20 W/ft2 of conditioned floor area.  
+**Rule Assertion:** P-RMR = Expected value   
+**Appendix G Section:** Section 3 Definitions      
+**90.1 Section Reference:** Section 3 Definitions  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Evaluate each Space Data Group
+
+**Applicability Checks:**  
+
+1. Applicable to spaces with the space.lighting_space_type equal to "COMPUTER_ROOM".
+
+**Function Calls:**  None  
+
+
+## Rule Logic:  
+**Applicability Check 1:**  
+- For each space in the P_RMR: `For space_p in P_RMR...Space:`
+    - Check if the lighting space type classification equals computer room: `if space_p.lighting_space_type == "COMPUTER_ROOM":`
+        - Reset the total space wattage variable to zero: `total_space_wattage = 0`
+        - For each miscellaneous equipment id in the space: `For misc_p in space_p.miscellaneous_equipment:`
+            - Check if it is IT equipment: `if misc_p.miscellaneous_equipment_type == "INFORMATION_TECHNOLOGY_EQUIPMENT":`
+                - Add Wattage to the total space Wattage: `total_space_wattage = total_space_wattage + misc_p.power`
+        - Get the square footage of the space: `floor_area = space_p.floor_area * CONVERSION(M2 to FT2)`
```

Comment:
I don't think we need "conversion" statement here as conversion of all RMD values will happen at the time when the RCT will parse the RMD.

Reply:
@mkarpman I agree that we don't need the conversion metric (i.e. 10.76) but we do need the conversion(m2 to ft2) to inform the RCT  team on what metrics to use.. The way CLP has it here looks good to me.

Reply:
@claperle based on the discussion last week, i take back my comment. We don't need the conversion statement here.